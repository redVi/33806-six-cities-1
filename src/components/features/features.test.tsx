import React from 'react';
import renderer from 'react-test-renderer';
import Features from '@/components/features/features';

describe(`Features`, () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(
      <Features type="room" maxAdults={2} bedrooms={2} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
