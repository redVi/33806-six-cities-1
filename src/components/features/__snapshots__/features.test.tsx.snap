// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Features renders correctly 1`] = `
<ul
  className="property__features"
>
  <li
    className="property__feature property__feature--entire"
  >
    Private Room
  </li>
  <li
    className="property__feature property__feature--bedrooms"
  >
    2
     
    Bedrooms
  </li>
  <li
    className="property__feature property__feature--adults"
  >
    Max 
    2
     adults
  </li>
</ul>
`;
